languages/rust/oso/examples/blog.rs

```diff
@@ -0,0 +1,142 @@
+//! Code for showing blog progress
```

Comment:
It will now with `--all-targets`. There's not really any need to have multiple makefiles since `cargo test` from the root will test all the rust crates